next.config.js

```diff
@@ -8,4 +8,13 @@ module.exports = withBundleAnalyzer({
   experimental: {
     missingSuspenseWithCSRBailout: false,
   },
+  async redirects() {
+    return [
+      {
+        source: '/',
+        destination: '/projects',
+        permanent: true, // Sends a 308 Permanent Redirect status code
+      },
+    ]
+  },
```

Comment:
![image](https://github.com/user-attachments/assets/0bd6fb10-4851-494c-949d-f95797943c7c)

Reply:
This is a way more seo friendly approach